src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttServerHandler.java

```diff
@@ -78,7 +79,7 @@ private void handleConnectMessage(ChannelHandlerContext ctx) {
      * @param publishMessage represents a MqttPublishMessage
      */
     private void handlePublishMessage(ChannelHandlerContext ctx, MqttPublishMessage publishMessage) {
-        logger.info("Topic: {}", publishMessage.variableHeader().topicName());
-        logger.info("Message: {}", publishMessage.payload().toString(Charset.defaultCharset()));
+        logger.info("MQTT Topic: {}", publishMessage.variableHeader().topicName());
+        logger.info("Mapped Topic: {}", mapper.map(publishMessage.variableHeader().topicName()));
```

Comment:
Is the "Mapped Topic" the name of the Kafka topic?

Reply:
the Mapped Topic is the result of the mapping, yes, it is the Kafka topic.

Reply:
Could we call it Kafka Topic instead?
```suggestion
        logger.info("Kafka Topic: {}", mapper.map(publishMessage.variableHeader().topicName()));
```

Reply:
+1